src/commands/lighthouse/show.ts

```diff
@@ -0,0 +1,33 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../../oclif/CustomCommand'
+import { showReports } from '../../modules/lighthouse/showReports'
+
+export default class ShowReports extends CustomCommand {
+  static description = 'Show previous saved audit reports, filtering by app and/or url'
+
+  static examples = [
+    'vtex lighthouse show -app=vtex.awesome-app',
```

Comment:
Maybe `--` is necessary?
```suggestion
    'vtex lighthouse show --app=vtex.awesome-app',
```

Reply:
Sure! Missed that :/